tests/common/devices/eos.py

```diff
@@ -105,7 +105,7 @@ def no_shutdown_multiple(self, interfaces):
     def check_intf_link_state(self, interface_name):
```

Comment:
the main issue I think is this function checks for admin status, not oper status you are looking for.

```
#show int status
Port       Name                                          Status       Vlan     Duplex  Speed    Type                     Flags Encapsulation
Et1        xxx-Ethernet1/1                        connected    2461     full         40G    40GBASE-CR4                    
 
#show int Et1
Ethernet1 is up, line protocol is up (connected)
```